src/core/InlineBlock.js

```diff
@@ -1,6 +1,6 @@
 import sqlTokenTypes from "./tokenTypes";
 
-const INLINE_MAX_LENGTH = 35;
+const INLINE_MAX_LENGTH = 50;
```

Comment:
I think we should also have tests for values that are shorter than 50 will stay on single line and longer ones get split onto multiple lines.